package.json

```diff
@@ -54,6 +54,7 @@
     "express": "^4.16.4",
     "gts": "^1.0.0",
     "is-docker": "^2.0.0",
+    "is-stream": "^2.0.0",
```

Comment:
This probably needs to be a regular (not dev) dependency :)

Reply:
Done